src/register/RegistrationPage.jsx

```diff
@@ -65,34 +65,25 @@ const RegistrationPage = (props) => {
     institutionLogin,
   } = props;
 
-  const {
-    registrationFormData: backedUpFormData,
-    registrationError,
-    registrationError: {
-      errorCode: registrationErrorCode,
-    } = {},
-    registrationResult,
-    shouldBackupState,
-    userPipelineDataLoaded,
-    submitState,
-    validations,
-  } = useSelector(state => state.register);
-
-  const {
-    fieldDescriptions,
-    optionalFields,
-    thirdPartyAuthApiStatus,
-    thirdPartyAuthContext,
-    thirdPartyAuthContext: {
-      autoSubmitRegForm,
-      errorMessage: thirdPartyAuthErrorMessage,
-      finishAuthUrl,
-      currentProvider,
-      providers,
-      secondaryProviders,
-      pipelineUserDetails,
-    },
-  } = useSelector(state => state.commonComponents);
+  const backedUpFormData = useSelector(state => state.register.registrationFormData);
+  const registrationError = useSelector(state => state.register.registrationError);
+  const registrationErrorCode = useSelector(state => state.register.registrationError?.errorCode);
```

Comment:
It would get default value `undefined` if errorCode is not available. In the original implementation, the value will be `{}` if errorCode is not available. Can you check if the following work?
```suggestion
const registrationErrorCode = useSelector(state => state.register.registrationError?.errorCode ?? {});
```

Reply:
In the original implementation, the default value is `{}` for `registrationError` not for `registrationErrorCode` and in our implementation the default value is `{}` too for `registrationError` but `undefined` for `registrationErrorCode` and it would be work fine because where the `registrationErrorCode` is using there is if condition in `useEffect` and it will handle the `undefined` case.